pymc4/distributions/mixture.py

```diff
@@ -19,11 +26,31 @@ class Mixture(Distribution):
     ----------
     p : array of floats
         p >= 0 and p <= 1
-        the mixture weights, in the form of probabilities
+        the mixture weights, in the form of probabilities, must sum to one.
     distributions : multidimensional PyMC4 distribution (e.g. `pm.Poisson(...)`)
         or iterable of one-dimensional PyMC4 distributions the
         component distributions :math:`f_1, \ldots, f_n`
     """
 
     def __init__(self, name, p, distributions, **kwargs):
         super().__init__(name, p=p, distributions=distributions, **kwargs)
+
+    @staticmethod
+    def _init_distribution(conditions, **kwargs):
+        p, d = conditions["p"], conditions["distributions"]
+        # if 'd' is a sequence of pymc distributions, then use the underlying
+        # tfp distributions for the mixture
+        if isinstance(d, (list, tuple)):
```

Comment:
You should use something like `collections.abc.Sequence` instead

Reply:
`collections.abc.Sequence` recognizes a string as a sequence, too. Shall I still change it?

Reply:
Yes, a string, even though it is a sequence, would raise a TypeError later because its elements aren't pm.Distribution instances